src/components/CautionaryContact/CautionaryContactTable.test.js

```diff
@@ -2,7 +2,7 @@ import { render } from '@testing-library/react'
 import UserContext from '../UserContext'
 import CautionaryContactTable from './CautionaryContactTable'
 import { operative } from 'factories/operative'
-import { CAUTIONARY_CONTACT } from '../../utils/cautContactAlertsDescription'
+import { CAUTIONARY_CONTACT } from '../../utils/cautContactAlerts'
```

Comment:
I know this wasn't named in this PR but would prefer this to be `CAUTIONARY_CONTACTS` to indicate we're working with an array.

Reply:
Yes I agree, we've discussed this again but due to the fact in designs were using singular @siapankina suggested to leave it the same. I think we should use plural and suggest to design team to update designs and have it everywhere plural. After discussing with Yasmeen we agreed on using `CAUTIONARY_ALERTS` due to the fact will include personal and location alerts